tce/src/applibs/LLVMBackend/plugin/TCEISelLowering.cc

```diff
@@ -939,6 +949,8 @@ TCETargetLowering::getRegForInlineAsmConstraint(
     case 'f':
```

Comment:
Confirmed!  Thank you very much for your help -- it would have taken a very long time for me to find this!